dojo/api_v2/serializers.py

```diff
@@ -1498,6 +1498,7 @@ class ImportScanSerializer(serializers.Serializer):
     product_type_name = serializers.CharField(required=False)
     product_name = serializers.CharField(required=False)
     engagement_name = serializers.CharField(required=False)
+    engagement_end_date = serializers.DateField(required=False)
```

Comment:
Can you add a help_text here to explain what the field does? Same for the Reimporter

Reply:
@coheigea are the help text enough now or did I understood the intention of them wrong?